main/webapp/modules/core/scripts/facets/list-facet.js

```diff
@@ -777,3 +785,162 @@ function tags_to_control(str) {
   }
   return str;
 }
+
```

Comment:
Wow, that's a serious amount of custom code, just for FF support!  Maybe this is a code smell that we are doing something wrong in our approach? Don't know, not an expert.

Reply:
Not sure though. As there are some plugins I tried them as well those were not working on FF due to this backspace issue.